safenode/src/protocol/types/dbc_genesis.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use sn_dbc::{
+    rng, Dbc, DbcTransaction, Error as DbcError, Hash, InputHistory, MainKey, RevealedAmount,
+    RevealedInput, Token, TransactionBuilder,
+};
+
+use std::fmt::Debug;
+use thiserror::Error;
+
+/// Amount of tokens to be owned by the Genesis DBC.
+/// At the inception of the Network a total supply of 4,294,967,295 whole tokens will be created.
```

Comment:
This shall actually be only 30% of it. Will fix the comment.

Reply:
The `GENESIS_DBC` will not be the full `u32::MAX`, but only 30% of it. 70% is to be issued later, when a solution to that has been found.
So, that's what needs an update here.